meta-sokol-flex-support/recipes-core/meta/archive-release.bb

```diff
@@ -741,6 +741,7 @@ do_archive_images () {
         echo "${WORKDIR}/xlayers.conf" >>include
     fi
 
+    sed -i 's//usr/bin/env python//usr/bin/env python3/g' "${WORKDIR}/bmaptool"
```

Comment:
I'd suggest using a different symbol than / for the sed fields separator as we already have /(s) in the path. i.e.
```
sed -i 's#/usr/bin/env python#/usr/bin/env python3#g' "${WORKDIR}/bmaptool"
```

Also, please update the commit title so we know what we're modifying exactly. I'd probably go with
```
archive-release:bmap-tools: use python3 for the shebang
```
but you can pick something else if you think that's more suitable.

Reply:
> I'd suggest using a different symbol than / for the sed fields separator as we already have /(s) in the path. i.e.
> 
> ```
> sed -i 's#/usr/bin/env python#/usr/bin/env python3#g' "${WORKDIR}/bmaptool"
> ```
> 
> Also, please update the commit title so we know what we're modifying exactly. I'd probably go with
> 
> ```
> archive-release:bmap-tools: use python3 for the shebang
> ```
> 
> but you can pick something else if you think that's more suitable. The commit message does not tell what the problem is in this case.

Made the suggested changes.